src/xdocs/config_filters.xml

```diff
@@ -2232,6 +2232,261 @@ public class UserService {
+              <td>Specify check ID pattern to suppress.</td>
+              <td><a href="property_types.html#Pattern">Pattern</a></td>
+              <td><code>null</code></td>
+              <td>10.8.2</td>
+            </tr>
+            <tr>
+              <td>lineRange</td>
+              <td>Specify negative/zero/positive value that defines the number of lines
+                preceding/at/following the suppressing nearby text. Property can
+                also be a RegExp group index at <code>nearbyTextPattern</code>
+                in format of <code>$x</code> and be picked from target file.</td>
+              <td><a href="property_types.html#String">String</a></td>
+              <td><code>"0"</code></td>
+              <td>10.8.2</td>
+            </tr>
+          </table>
+        </div>
+      </subsection>
+      <subsection name="Examples" id="SuppressWithNearbyTextFilter_Examples">
+        <p>
+          To configure the filter to suppress audit events on the same line:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;/&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;MagicNumber&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example:
+        </p>
+        <source>
+int a = 1; // SUPPRESS CHECKSTYLE the reason for this is..
+int b = 1; // violation
+        </source>
+        <p>
+          To configure the filter to suppress audit events on any line that contains
+          <code>DO NOT CHECK THIS LINE</code>:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;&gt;
+    &lt;property name=&quot;nearbyTextPattern&quot; value=&quot;DO NOT CHECK THIS LINE&quot;/&gt;
+  &lt;/module&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;MagicNumber&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example:
+        </p>
+        <source>
+int a = 1; // DO NOT CHECK THIS LINE
+int b = 1; // violation
+        </source>
+        <p>
+           To configure the filter to suppress audit events whose check message contains
+            the word <code>Line</code>. In this case, <code>LineLengthCheck</code>'s violation
+            message contains it:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;&gt;
+    &lt;property name=&quot;messagePattern&quot; value=&quot;.*Line.*&quot;/&gt;
+    &lt;property name=&quot;nearbyTextPattern&quot; value=&quot;.*&quot;/&gt;
+  &lt;/module&gt;
+  &lt;module name=&quot;LineLength&quot;&gt;
+    &lt;property name=&quot;max&quot; value=&quot;10&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example:
+        </p>
+        <source>
+export FOO=BAR # ok, even though line is 14 chars
```

Comment:
ok, because violation message is matching suppress pattern

Reply:
Done.